src/session.rs

```diff
@@ -897,9 +881,7 @@ where
 
         self.flush(cx);
 
-        let mut is_pending = self.poll_inner_socket(cx).is_pending();
```

Comment:
Why removing this line?

Reply:
Maybe you haven't read the content of pr completely, the socket is spawned separately, this field no longer exists